src/mbgl/annotation/point_annotation_impl.cpp

```diff
@@ -13,11 +13,11 @@ void PointAnnotationImpl::updateLayer(const TileID& tileID, AnnotationTileLayer&
     featureProperties.emplace("sprite", point.icon.empty() ? std::string("default_marker") : point.icon);
 
     const uint16_t extent = 4096;
-    const mbgl::PrecisionPoint pp = point.position.project();
```

Comment:
This code was odd to begin with and should probably be replaced wholesale with something that makes more sense. It shouldn't round trip through screen coordinates, for one thing. And stuff like `pp.x * z2 - x`, where `x = pp.x * z2`, looks nonsensical.